plugins/shared-react/src/types/pipeline/taskRun.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  V1ConfigMap,
+  V1ObjectMeta,
+  V1PersistentVolumeClaimTemplate,
+  V1Secret,
+} from '@kubernetes/client-node';
+import { RunStatus } from '@patternfly/react-topology';
```

Comment:
do we need to have `@patternfly/react-topology` as dependency to shared package?